packages/@uppy/companion/src/server/controllers/s3.js

```diff
@@ -353,7 +357,60 @@ module.exports = function s3 (config) {
     }, next)
   }
 
+  const policy = {
+    Version: '2012-10-17',
+    Statement: [
+      {
+        Effect: 'Allow',
+        Action: [
+          's3:PutObject',
+        ],
+        Resource: [
+          `arn:aws:s3:::${config.bucket}/*`,
+          `arn:aws:s3:::${config.bucket}`,
+        ],
+      },
+    ],
+  }
+
+  let stsClient
+  function getSTSClient () {
+    if (stsClient == null) {
+      stsClient = new STSClient({
+        region: config.region,
+        credentials : {
+          accessKeyId: config.key,
+          secretAccessKey: config.secret,
+        },
+      })
+    }
+    return stsClient
+  }
```

Comment:
```suggestion
  const stsClient = new STSClient({
    region: config.region,
    credentials : {
      accessKeyId: config.key,
      secretAccessKey: config.secret,
    },
  })
```

Reply:
Hum I don't think that's a good idea, it would make Companion fail to load for all users who have not updated the permissions, making it quite the breaking change.